src/services/axios.ts

```diff
@@ -1,11 +1,21 @@
 import axios from 'axios'
-import { setupCache } from 'axios-cache-interceptor'
+import { buildKeyGenerator, setupCache } from 'axios-cache-interceptor'
 
 import { addTrailingSlashToUrl } from './url'
 
 const baseURL = __DEV__ ? 'https://lunes-test.tuerantuer.org/api' : 'https://lunes.tuerantuer.org/api'
 
-setupCache(axios)
+const keyGenerator = buildKeyGenerator(true, ({ headers, baseURL = '', url = '', method = 'get', params, data }) => ({
+  url: baseURL + (baseURL && url ? '/' : '') + url,
+  headers: headers?.Authorization ?? 'not-set',
+  method,
+  params: params as unknown,
```

Comment:
Why `as unknown`? Is it not possible to correctly type this?

Reply:
No I guess not. You can not know what is going to be passed for params. It's the same code as the defaultKeyGenerator form axios-cache-interceptor uses.